scripts/makeSampleMeshFiles.py

```diff
@@ -265,14 +264,26 @@ def generate_all_types(n):
                         type=int,
                         nargs='+',
                         help='an integer side length for mesh construction')
-    parser.add_argument('--seed',
+
+    parser.add_argument("-p",
+                        "--perturbation",
+                        dest="perturbation_size",
+                        type=float,
+                        default=0.0,
+                        help='maximu size of the perturbation to generate non rectilinear mesh. default = 0')
+
+    parser.add_argument("-sd",
+                        "--seed",
+                        dest="seed",
                         type=int,
                         nargs='?',
                         const=5,
                         default=5,
-                        help='Integer random number generator seed, default=5')
+                        help='Integer random number generator seed, default=5')                                        
```

Comment:
It sounds like the random seed isn't guaranteed to produce the same results across versions of numpy/platforms, FYI.  If you are trying to limit diffs in the repo, it may be worth noting these somewhere

Reply:
I didn't really crate the original script but I am guessing the idea was to be able to reproduce the same meshes.